platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheIncompatibleTasksIntegrationTest.groovy

```diff
@@ -33,8 +33,8 @@ class ConfigurationCacheIncompatibleTasksIntegrationTest extends AbstractConfigu
         result.assertTasksExecuted(":declared")
         fixture.assertStateStoredAndDiscarded {
             hasStoreFailure = false
-            problem "Build file 'build.gradle': line 16: invocation of 'Task.project' at execution time is unsupported."
```

Comment:
What happened to the line numbers?

Reply:
The logic fow how we determine a stack portion to search for a location changed subtly, and the build script frame did not make it in there. I initially thought (looking at the required test changes) that it's a win -- to be able to attribute a problem to an owner like a task. But now I see this is a degradation.

I patched that up. Thank you for noticing!